client/ayon_unreal/plugins/create/create_render.py

```diff
@@ -51,7 +52,7 @@ def create_with_new_sequence(
         # If the option to create a new level sequence is selected,
         # create a new level sequence and a master level.
 
-        root = "/Game/Ayon/Sequences"
+        root = f"{HALON_PATH_CONFIG}/Sequences"
```

Comment:
This feels a bit hardcoded, try adding a setting for sequence root or something to that effect in the settings and use that instead